docs/core/porting/third-party-deps.md

```diff
@@ -63,30 +68,59 @@ dotnet5.4
 dotnet5.5
 ```
 
-**While these will likely work with your code, there is no guarantee of compatibility**.  Packages with these TFMs were built with pre-release .NET Core packages.  Take note of when (or if) packages like this are updated to be `netstandard`-based.
+While these TFMs likely work with your code, there is no guarantee of compatibility. Packages with these TFMs were built with pre-release .NET Core packages. Take note of when (or if) packages using these TFMs are updated to be .NET Standard-based.
 
 > [!NOTE]
-> To use a package targeting a traditional PCL or pre-release .NET Core target, you must use the `imports` directive in your `project.json` file.
+> To use a package targeting a traditional PCL or pre-release .NET Core target, you must use the `PackageTargetFallback` MSBuild element in your project file.
+> For more information about this MSBuild element, see [`PackageTargetFallback`](../tools/csproj.md#packagetargetfallback).
```

Comment:
no paragraph break.

Reply:
There's no paragraph break here (I'd need an extra line in Markdown to create that, so keeping as-is)

![image](https://user-images.githubusercontent.com/12971179/35716205-1f36b98c-078c-11e8-9ef4-415e1c1f5934.png)